psalm.xml

```diff
@@ -131,6 +151,9 @@
                 <!-- See https://github.com/doctrine/dbal/pull/3562 -->
                 <file name="lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php"/>
                 <file name="lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php"/>
+
+                <!-- See https://github.com/doctrine/dbal/pull/4347 -->
```

Comment:
It doesn't explain why this suppression is being introduced. All other referenced GitHub pull requests and issues actually fix those suppressed violations.

Reply:
That particular line isn't needed anymore, I figured. I'll see what I can do about the other ones.

Reply:
Check again, please.

Reply:
Removing is fine.